Ladybird/AppKit/UI/Inspector.h

```diff
@@ -11,15 +11,19 @@
 @class LadybirdWebView;
 @class Tab;
 
-@interface Inspector : NSObject
+@interface Inspector : NSScrollView
```

Comment:
Can we make the Inspector be the scroll view in the previous commit? Changing it to an NSObject in one commit then NSScrollView in the next commit is a bit churny.